runtime/src/bank.rs

```diff
@@ -1761,4 +1770,42 @@ mod tests {
         assert!(!bank5.is_in_subtree_of(2));
         assert!(!bank5.is_in_subtree_of(4));
     }
+
+    #[test]
+    fn test_bank_inherit_tx_count() {
+        let (genesis_block, mint_keypair) = GenesisBlock::new(500);
+        let parent = Arc::new(Bank::new(&genesis_block));
+
+        // Bank 1
+        let bank = Arc::new(new_from_parent(&parent));
+        // Bank 2
+        let bank2 = Arc::new(new_from_parent(&bank));
+        // Bank 5
+        let bank5 = new_from_parent(&bank);
+
+        let key1 = Keypair::new();
+
+        // move a token
+        let tx_move_mint_to_1 =
```

Comment:
In the other tests, `tx` is a suffix, not a prefix. Also, `move` should be `transfer`.